apps/designer/app/designer/features/style-panel/controls/toggle/toggle-control.tsx

```diff
@@ -0,0 +1,41 @@
+import { Flex, ToggleGroup, Tooltip } from "@webstudio-is/design-system";
+import { StyleProperty } from "@webstudio-is/react-sdk";
+import { useIsFromCurrentBreakpoint } from "../../shared/use-is-from-current-breakpoint";
+
+export type ToggleGroupControlProps = {
+  property: StyleProperty | StyleProperty[];
+  value: string;
+  items: { child: JSX.Element; label: string; value: string }[];
+  onValueChange?: (value: string) => void;
+};
+
+export const ToggleGroupControl = ({
+  property,
+  value = "",
+  items = [],
+  onValueChange,
+}: ToggleGroupControlProps) => {
+  const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  return (
+    <ToggleGroup.Root
```

Comment:
Lets export the Toggle group in the ds as `export const ToggleGroupRoot` ... we have it currently inconsistent, but I think this is better ... alternatively the way you did it here and then renaming the rest